packages/client/src/components/PlayButton/button.module.css

```
@@ -0,0 +1,32 @@
+.circle,
+.button {
+  border-radius: 50%;
+}
+
+.button {
+  border: none;
+  background-color: var(--color-accent-tertiary);
+  position: relative;
+  width: 150px;
+  height: 150px;
+  cursor: pointer;
+}
+
+.circle {
+  position: absolute;
+  border: 2px solid var(--color-accent-secondary);
+  top: -11px;
+  left: -1px;
+  width: 148px;
+  height: 148px;
+}
+
+.button svg {
+  color: var(--color-accent-secondary);
+  width: 48px;
```

Контейнер иконки должен быть 70 х 70.

Вообще, я бы блок с иконкой сделал немного иначе. Если поместить его в circle и центрировать внутри (так как иконка должна быть в центре относительно круга с обводкой). Это позволит, во-первых, применить более безопасное центрирование, во-вторых, избавиться от абсолютного позиционирования в иконке, и соответственно со свойствами top, left, transform. 
Останется только ширину и высоту ей выставить нужную

```suggestion
  width: 70px;
```